client/src/components/Offer/Offer.module.scss

```
@@ -0,0 +1,35 @@
+@use './helpers/stylesHelpers' as helper;
+
+.new {
+  border-left: helper.$newIconColor 3px solid;
+}
+
+.offer {
+  position: relative;
+  background-color: rgb(255, 255, 255);
+  padding: 1rem;
+  margin: 0.5rem;
+  display: flex;
+  flex-direction: row;
+  box-shadow: 0 4px 8px 0 rgb(0 0 0 / 10%), 0 6px 20px 0 rgb(0 0 0 / 5%);
+  border-radius: 2px;
+  color: rgba(0, 0, 0, 0.733);
+  text-decoration: none;
+  transition: 0.3s;
+  max-width: 1000px;
+  min-height: 122px;
+}
+
+.offer:hover {
+  background-color: rgba(229, 229, 245, 0.199);
+  transition: 0.3s;
+}
```

You could use scss feature to nest this:
```suggestion
.offer {
  position: relative;
  background-color: rgb(255, 255, 255);
  padding: 1rem;
  margin: 0.5rem;
  display: flex;
  flex-direction: row;
  box-shadow: 0 4px 8px 0 rgb(0 0 0 / 10%), 0 6px 20px 0 rgb(0 0 0 / 5%);
  border-radius: 2px;
  color: rgba(0, 0, 0, 0.733);
  text-decoration: none;
  transition: 0.3s;
  max-width: 1000px;
  min-height: 122px;
  
  &:hover {
    background-color: rgba(229, 229, 245, 0.199);
    transition: 0.3s;
  }
}
```